docs/src/api.md

```diff
@@ -8,10 +8,11 @@ In this page we list the remaining API functions, which constitute the bulk of A
 - [`AgentBasedModel`](@ref)
 - [`StandardABM`](@ref)
 - [`EventQueueABM`](@ref)
+- [`step!(::Agent, args...)`](@ref): progress the model forwards in time.
```

Comment:
Sorry, I was not clear. I meant to include this bullet point within the docstring of `AgentBasedModel` where it illustrates many other fundamental API functions like `model[id`].

Reply:
Done, @Datseris